docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<p>Somewhat off-topic questions: How does <code>nfe=250</code> result in 297 iterations? Is this a stochastic process?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>The stopping condition is set to 250 nfe. But the algorithm is population-based so will only stop after the last population has been evaluated and thus frequently your real nfe will be slightly higher than the stopping condition.</p>